test/NuGet.Core.Tests/NuGet.Commands.Test/MSBuildRestoreUtilityTests.cs

```diff
@@ -3829,6 +3829,113 @@ public void MSBuildRestoreUtility_GetPackageSpec_CPVM_VersionOverrideCanBeDisabl
             }
         }
 
+        [Theory]
+        [InlineData(null, false)]
+        [InlineData("", false)]
+        [InlineData("                     ", false)]
+        [InlineData("false", false)]
+        [InlineData("invalid", false)]
+        [InlineData("true", true)]
+        [InlineData("           true    ", true)]
```

Comment:
would it be more in the spirit of unit testing to make `MSBuildRestoreUtility.IsPropertyTrue` internal, then have a unit test validating whitespace is trimmed correctly, so that tests titled "CPVM floating versions can be enabled", so we don't need so many duplicate tests for a test whose named resposibility is not validating `IsPropertyTrue` edge cases?

Reply:
Good idea, done!